lib/nfs_common.pm

```diff
@@ -277,9 +277,16 @@ sub check_y2_nfs_func {
     stop_service();
     # we need to cleanup the nfs settings after service check was done.
     if ($stage eq 'after') {
-        zypper_call 'rm yast2-nfs-server nfs-kernel-server', timeout => 480;
-        script_run ':> /etc/exports';
+        zypper_call 'rm yast2-nfs-server', timeout => 480;
+        # remove added nfs entry from /etc/fstab and /etc/exports
+        assert_script_run 'sed -i \'/srv/d\' /etc/fstab';
+        assert_script_run 'sed -i \'/srv/d\' /etc/exports';
         script_run 'rm -fr /srv/*';
+        script_run 'rm -fr /tmp/nfs';
+        # we need to restart rpcbind and rpcbind.socket for rpcbind test
+        systemctl('restart nfs-server',     ignore_failure => 1);
+        systemctl('restart rpcbind',        ignore_failure => 1);
+        systemctl('restart rpcbind.socket', ignore_failure => 1);
```

Comment:
It's better to verify the status of these services using systemctl('is-active' xxx);